spec/models/item_spec.rb

```diff
@@ -1,11 +1,34 @@
 require 'rails_helper'
 
-RSpec.describe Item, type: :model do
-  it 'scopes by name' do
-    item1 = create(:item, name: "Softie Jam")
-    item2 = create(:item, name: "Softie Marmelade")
+describe Item, type: :model do
+  describe 'name scoping' do
+    it 'should find matching query' do
+      item = create(:item, name: "Softie Jam")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.